packages/devextreme/js/ui/menu.d.ts

```diff
@@ -105,8 +105,15 @@ export type SelectionChangedEvent<TKey = any> = EventInfo<dxMenu<TKey>> & Select
  * @inherits EventInfo
  */
 export type SubmenuHiddenEvent<TKey = any> = EventInfo<dxMenu<TKey>> & {
+    /**
+     * @docid _ui_menu_SubmenuHiddenEvent.itemData
+     * @type object
+     */
+    readonly itemData?: MenuBasePlainItem;
```

Comment:
There are also `dxMenuBaseItem` and `dxMenuItem` types, please check which one should be here